netbox/extras/tables/tables.py

```diff
@@ -1,8 +1,7 @@
-import json
-
 import django_tables2 as tables
 from django.conf import settings
 
+import json
```

Comment:
The `json` import should be at top as it's a standard Python library.

```suggestion
import json

import django_tables2 as tables
from django.conf import settings

```